### PR TITLE
Story 1 task 1

### PR DESCRIPTION
**Pull request by tauroid**
WAVE has a contrast issue but it's the exact colour from the wireframe.. maybe we could ask the PO about changing it?

### CODE REVIEW

File: src/Navbar/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+function Navbar() {
+  return (
+    <nav className="navbar navbar-expand border-bottom shadow-sm px-4 justify-content-md-start justify-content-center">
+        <a href="#" className="navbar-brand fw-bold m-0">
```
Comment (robin-hodge) [inline src/Navbar/Navbar.js:4]: I'm not sure why this is a link?

File: src/Main/Main.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import './Main.scss'
+import burgerImage from './burgers.webp'
+import Navbar from '../Navbar/Navbar'
+import BannerBoxFoodDelivered from '../BannerBoxFoodDelivered/BannerBoxFoodDelivered'
+
+function Main() {
+  return (
+    <main className="Main">
```
Comment (robin-hodge) [inline src/Main/Main.js]: the classname is created but not used in the styling file, so I wonder why you have used it?

File: src/Main/Main.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      <Navbar />
+      <div style={{backgroundImage: "url('" + burgerImage + "')"}} className="burger-image m-md-3">
+        <div className="row justify-content-center align-items-center h-100 m-3">
```
Comment (robin-hodge) [inline src/Main/Main.js]: as we understand it, rows must live inside a div/element with a container class https://stackoverflow.com/questions/19063976/bootstrap-grid-do-i-need-a-container

File: src/Main/Main.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import './Main.scss'
+import burgerImage from './burgers.webp'
+import Navbar from '../Navbar/Navbar'
+import BannerBoxFoodDelivered from '../BannerBoxFoodDelivered/BannerBoxFoodDelivered'
+
+function Main() {
+  return (
+    <main className="Main">
+      <Navbar />
+      <div style={{backgroundImage: "url('" + burgerImage + "')"}} className="burger-image m-md-3">
```
Comment (robin-hodge) [inline src/Main/Main.js:10]: are the double {} necessary here?

File: src/Main/Main.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+        <div className="row justify-content-center align-items-center h-100 m-3">
+          <div className="col-sm-12 col-md-5 semi-transparent bg-light h-50 flex-row justify-content-center align-items-center d-flex m-0">
```
Comment (robin-hodge) [inline src/Main/Main.js]: there is a point in the level of responsiveness where the text 'Order your favourite food from local restaurants, right to your door.' spills underneath the semi-transparent box,

Comment (robin-hodge) [reply src/Main/Main.js]: 845px seems to be the breakpoint!

File: src/Main/Main.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+            <BannerBoxFoodDelivered />
```
Comment (robin-hodge) [inline src/Main/Main.js]: change as discussed

File: src/Main/Main.js (hunk 2 above)
Comment (DanAbsolon) [reply src/Main/Main.js]: agreed

File: src/Main/Main.js (hunk 5 above)
Comment (DanAbsolon) [reply src/Main/Main.js]: We've changed the col-md-5 to col-md-6 - hopefully that's fixed it but we can't test it here as we aren't getting the problem. Can you test it again at your end to see if it.worked?

Comment (robin-hodge) [reply src/Main/Main.js]: 875 breakpoint!

**Review by robin-hodge — CHANGES_REQUESTED** [review]
Just the breakpoint please!

**Review by robin-hodge — APPROVED** [review]

**Pull request merged**